File: .sdk/tm/csharp/test/ExistsTest.cs
// BluefinDecryptx SDK exists test.

using Xunit;

using BluefinDecryptxSdk;

namespace BluefinDecryptxSdk.Test;

public class ExistsTest
{
    [Fact]
    public void TestMode()
    {
        var testsdk = BluefinDecryptxSDK.TestSDK(null, null);
        Assert.NotNull(testsdk);
    }
}

File: rb/test/exists_test.rb
# BluefinDecryptx SDK exists test

require "minitest/autorun"
require_relative "../BluefinDecryptx_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = BluefinDecryptxSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
